plugins/jobs/src/js/pages/JobDetailPage.js

```diff
@@ -0,0 +1,193 @@
+import classNames from "classnames";
+import prettycron from "prettycron";
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+import { routerShape } from "react-router";
+import mixin from "reactjs-mixin";
+
+import JobFormModalContainer from "#PLUGINS/jobs/src/js/JobFormModalContainer";
+import TaskStates from "#PLUGINS/services/src/js/constants/TaskStates";
+
+import Icon from "#SRC/js/components/Icon";
+import Page from "#SRC/js/components/Page";
+import TimeAgo from "#SRC/js/components/TimeAgo";
+import TabsMixin from "#SRC/js/mixins/TabsMixin";
+import StringUtil from "#SRC/js/utils/StringUtil";
+import Job from "#SRC/js/structs/Job";
+import JobConfiguration from "./JobConfiguration";
+import { DIALOGS } from "./JobDetailPageContainer";
+import JobRunHistoryTable from "./JobRunHistoryTable";
+
+class JobDetailPage extends mixin(TabsMixin) {
```

Comment:
🎨 Is it possible to relatively easy 🔥that mixin?

Reply:
I don't know and this is really out of scope for this PR, IMHO